src/WebJobs.Script/Environment/EnvironmentExtensions.cs

```diff
@@ -667,7 +667,7 @@ public static HashSet<string> GetLanguageWorkerListToStartInPlaceholder(this IEn
         {
             string placeholderList = environment.GetEnvironmentVariableOrDefault(RpcWorkerConstants.FunctionWorkerPlaceholderModeListSettingName, string.Empty);
             var placeholderRuntimeSet = new HashSet<string>(placeholderList.Trim().Split(new[] { ';' }, StringSplitOptions.RemoveEmptyEntries).Select(p => p.Trim()));
-            string workerRuntime = environment.GetEnvironmentVariable(RpcWorkerConstants.FunctionWorkerRuntimeSettingName);
+            string workerRuntime = environment.GetEnvironmentVariable(FunctionWorkerRuntime);
```

Comment:
Can you use the `IEnvironment` extension method `GetFunctionsWorkerRuntime`?

Reply:
I'm planning on getting rid of that in a subsequent PR.